src/lib/map/MapVeuContainer.tsx

```diff
@@ -20,12 +20,12 @@ import {
 } from '../core/hooks/client';
 
 import './MapVEu.scss';
-import { SemiTransparentHeaderLogoProps } from './analysis/SemiTransparentHeader';
+import { SiteProps } from './analysis/MapNavigation';
```

Comment:
I renamed these props because they're used for both the header and the lefthand side menu. These props do the following: https://github.com/VEuPathDB/web-eda/pull/1671/files?diff=split&w=1#r1123744372